doc/orchestrator.rst

```diff
@@ -1,60 +1,324 @@
+AI-enabled workflows including online training, low-latency inference, cross-application data
+exchange, online interactive visualization, online data analysis, computational
+steering, and more. The ``Orchestrator`` can be thought of as a general
+feature store capable of storing numerical data, ML models, and scripts.
+The orchestrator is capable of performing inference and script evaluation using data in the feature store.
+Any SmartSim ``Model`` or ``Ensemble`` model can connect to the
+``Orchestrator`` via the :ref:`SmartRedis<smartredis-api>`
+client library to transmit data, execute ML models, and execute scripts.
+
+SmartSim offers two types of orchestrator deployments: :ref:`clustered deployment<Clustered Deployment>` and
+:ref:`colocated deployment<Colocated Deployment>`. Continue to the respective deployment sections for
+information on deployment types.
+
+Below is a diagram demonstrating an orchestrator as a general feature store
+capable of storing numerical data (tensors and datasets), AI models, and scripts (TorchScript).
+Combined with the SmartRedis clients, the ``Orchestrator`` is capable of hosting and executing
+AI models written in Python on CPU or GPU. The ``Orchestrator`` supports models written with
+TensorFlow, Pytorch, TensorFlow-Lite, or models saved in an ONNX format (e.g. sci-kit learn).
 
 .. |orchestrator| image:: images/Orchestrator.png
   :width: 700
   :alt: Alternative text
 
 |orchestrator|
 
-Combined with the SmartRedis clients, the ``Orchestrator`` is capable of hosting and executing
-AI models written in Python on CPU or GPU. The ``Orchestrator`` supports models written with
-TensorFlow, Pytorch, TensorFlow-Lite, or models saved in an ONNX format (e.g. sci-kit learn).
-
-
-Cluster Orchestrator
-====================
-
-The ``Orchestrator`` supports single node and distributed memory settings. This means
-that a single compute host can be used for the database or multiple by specifying
-``db_nodes`` to be greater than 1.
+======================
+Clustered Deployment
+======================
+--------
+Overview
+--------
+During clustered deployment, a SmartSim ``Orchestrator`` (the database) runs on separate
+compute node(s) from the model node(s). A clustered orchestrator can be deployed on a single
+node or sharded (distributed) over multiple nodes.
+With a sharded orchestrator, available hardware for inference and script
+evaluation increases and overall memory for data storage increases.
+
+Communication between a clustered Orchestrator and Model
+is initialized in the application script via a SmartRedis client.
+Users do not need to know how the data is stored in a clustered configuration and
+can address the cluster with the SmartRedis clients like a single block of memory
+using simple put/get semantics in SmartRedis. The client establishes a
+connection using the database address and travels off the model compute node to
+the database compute node.
```

Comment:
As a reader, I'm having a hard time understanding what ".. and travels off the model compute node to the database compute node." means? 

Are we saying "the client establishes a connection and seamlessly handles communication with nodes in the cluster"?

Reply:
I was attempting to say that the client is initialized on the model node, and then travels off the node to the database nodes further emphasizing that the model and orch do not share compute resources but this might be more of a developer explanation and not something the user needs to worry about